src/main/java/com/teamc/fms/entity/Facility.java

```diff
@@ -0,0 +1,27 @@
+package com.teamc.fms.entity;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+import java.util.List;
+
+@Entity
+@Table(name = "facility")
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Facility {
+    @Id
+    private String id;
+    @NotNull
+    private String name;
+    @NotNull
```

Comment:
desciption is nullable
![image](https://github.com/web-development-course-way/fms/assets/31379298/9c48259d-623b-4a1a-b18d-8fc79f7361ec)

Reply:
Okay I'm gonna chnage it